File: examples/access-levels-write/access-levels-write.py
import json
import iam_floyd as statement

s = (
    # doc-start
    statement.Ec2() \
    .allow() \
    .all_write_actions()
    # doc-end
).to_json()

pretty = json.dumps(s, indent=4, sort_keys=True)
print(pretty)

File: examples/access-levels-write/access-levels-write.ts
import { deploy, out } from '../../helper/typescript/typescript_test';
import * as statement from '../../lib';

function getStatement() {
  return (
    // doc-start
    new statement.S3() //
      .allow()
      .allWriteActions()
    // doc-end
  );
}
const s = [getStatement()];
out(s);
deploy(s);
